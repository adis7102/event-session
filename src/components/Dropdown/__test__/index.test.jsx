import React from "react";
import Dropdown from '../index';

import { mount } from "enzyme";

it('should render Dropdown correctly', () => {
    const wrapper = mount(<Dropdown />);

    expect(wrapper).toMatchSnapshot();
});