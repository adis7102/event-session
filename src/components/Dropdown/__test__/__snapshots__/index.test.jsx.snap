// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dropdown correctly 1`] = `ReactWrapper {}`;
